htdocs/core/lib/functions.lib.php

```diff
@@ -7294,10 +7294,20 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
 	if (empty($exclude) || !in_array('objectamount', $exclude)) {
 		include_once DOL_DOCUMENT_ROOT.'/core/lib/functionsnumtoword.lib.php';
 
+		if (is_object($object)) {
+			$sumpayed = $object->getSommePaiement();
+			$sumdeposit = $object->getSumDepositsUsed();
+			$sumcreditnote = $object->getSumCreditNotesUsed();
+			$already_payed_all = $sumpayed + $sumdeposit + $sumcreditnote;
+		}
+
 		$substitutionarray['__DATE_YMD__']        = is_object($object) ? (isset($object->date) ? dol_print_date($object->date, 'day', 0, $outputlangs) : null) : '';
 		$substitutionarray['__DATE_DUE_YMD__']    = is_object($object) ? (isset($object->date_lim_reglement) ? dol_print_date($object->date_lim_reglement, 'day', 0, $outputlangs) : null) : '';
 
 		$substitutionarray['__AMOUNT__']          = is_object($object) ? $object->total_ttc : '';
+		$substitutionarray['__AMOUNT_FORMATED__']          = is_object($object) ? ($object->total_ttc ? price($object->total_ttc, 0, $outputlangs, 0, 0, -1, $conf->currency) : null) : '';
```

Comment:
If price is "formated", you should use -1 as parameter 5th parameter

Reply:
The error on travis may come because you make a 
$object->getSommePaiement();
$object->get...
So you call method that does not always exists for some objects. 
Try to test that method exists on object before calling them.

Reply:
> If price is "formated", you should use -1 as parameter 5th parameter

I've copied the line from version 13 working dolibarr installation.  I'm sorry not using the most current version, fixed in the commit. 

> The error on travis may come because you make a $object->getSommePaiement(); $object->get... So you call method that does not always exists for some objects. Try to test that $object->element == 'facture' before calling them so you are sure methods exists.

Used "instanceof" to better support polymorphism, fixed in the commit